OmniBLE/PumpManagerUI/ViewControllers/DashUICoordinator.swift

```diff
@@ -186,7 +186,7 @@ class DashUICoordinator: UINavigationController, PumpManagerOnboarding, Completi
                 self?.navigateTo(.deactivate)
             }
             
-            let view = hostingController(rootView: PairPodView(viewModel: viewModel))
+            let view = hostingController(rootView: PairPodView(viewModel: viewModel).onAppear(perform: {UIApplication.shared.isIdleTimerDisabled = true}), onDisappear: {UIApplication.shared.isIdleTimerDisabled = false})
```

Comment:
Normally the onAppear and onDisappear view modifiers should be attached to the views themselves.  I also see that you're not using a view modifier for onDisappear, but are using the DismissibleHostingController attribute instead.  What is the reason for that?

If we can't use the normal view modifiers, then maybe this should go into the `hostingController` function as a `disableIdleTimer` flag, and it can add the callbacks in one place, rather than having them repeated.

Also, I'm wondering if this should be limited to screens that we do expect people to have to be spending some time outside the app, like attaching the pod, waiting for the cannula insertion, etc.  The check cannula and setup complete views are generally short interaction screens.

Reply:
I didn‘t want to touch multiple files, so I attached those ViewModifiers to where the views are being initialized in the controller. 

I added `onDismiss` to the method head as param to the `hostingController` because I saw it was already there. 

I‘m sure we *can* do it in the views directly rather than in the `DashUICoordinator`- the issue is this cannot be tested in a simulator and testing on a real phone isn‘t as easy. 

I can make it a flag and handle it within the `hostingController`, no problem. The idea was to disable autolock for the entire process of changing a pod, but I can limit it to just pairing and inserting. 

I went by what various users told me, one saying 

> My son is not unique among kids in having anxiety around cannula insertion, and at times has used every delaying tactic under the sun. He's getting much better now, but when we first started trialing pods we would go through the screen lock timing out a dozen or more times with each pod, and each time that happens it resets his internal countdown to when he's ready, and jacks up the anxiety another notch. 

This does speak about insertion only but I thought it would help for the phone not to lock until the entire process is done, especially for caregivers changing pods on children.

Reply:
@ps2 I've reworked my previous commit with your points in mind. Hope that's better now 😄

Reply:
Thanks! Can you report on how much testing you've done?